librz/include/rz_analysis.h

```diff
@@ -737,6 +739,18 @@ typedef struct rz_analysis_var_t {
 	int argnum;
 } RzAnalysisVar;
 
+/**
+ * \brief Global variables
+ */
+typedef struct rz_analysis_var_global_t {
+	RBNode rb; ///< RBTree node for address management
+	char *name; ///< name of the variable
+	ut64 addr; ///< address of the global variable
+	RzType *type; ///< type of the variable
+	ut64 size; ///< size of the variable
```

Comment:
The size is uniquely defined by the type already, so this field should not exist.

Reply:
`RzType` does not have a size field.

Reply:
There is `rz_type_size()` function.

Reply:
Yeah, I use `rz_type_db_get_bitsize` to set the size field. I think this is reasonable.

Reply:
Please note, that `rz_type_db_get_bitsize()` recalculates the new size depending on the chosen `asm.arch`, `asm.bits`, `asm.cpu`, `asm.os`, etc